ydb/core/security/ticket_parser_impl.h

```diff
@@ -995,6 +1022,19 @@ class TTicketParserImpl : public TActorBootstrapped<TDerived> {
         return std::move(errorMessage);
     }
 
+    static TString GetErrorMessage(nebius::iam::v1::AuthorizeResult::ResultCode resultcode) {
+        switch (resultcode) {
```

Comment:
maybe it could be easier to just resolve enum to string, strip part of it and return as it is?